spec/controllers/expenses_controller_spec.rb

```diff
@@ -0,0 +1,273 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ExpensesController, type: :request do
+  let!(:current_user) { create(:user) }
+
+  let(:account) { create(:account) }
+  let(:category) { create(:category) }
+
+  let!(:current_expense) { create(:expense, name: 'Expense', value: 10, user_id: current_user.id) }
+  let!(:second_expense) { create(:expense) }
+
+  let(:jwt_token) { JsonWebToken.encode(user_id: current_user.id) }
+  let(:json) { JSON.parse(response.body) }
+
+  describe '#index' do
+    subject { get(expenses_path, headers: { Authorization: jwt_token }) }
+
+    it 'returns all expenses from current user' do
+      subject
+
+      expect(response).to have_http_status :ok
+      expect(json.count).to eq(1)
+      expect(json[0]['name']).to include('Expense')
+      expect(json[0]['value_in_cents']).to eq(1000)
+      expect(Time.zone.parse(json[0]['paid_at'])).to eq(Time.parse('28-04-2022'))
```

Comment:
Será que algo assim funcionaria?
```suggestion
      expect(json[0]['paid_at']).to eq(Time.parse('28-04-2022').iso8601)
```

Reply:
Ou põe a data direto em string, exemplo:
```suggestion
      expect(json[0]['paid_at']).to eq("2022-04-28T00:00:00+00:00")
```